main/guides/zoe/contract-basics.md

```diff
@@ -25,7 +25,7 @@ Putting it all together:
 Let us save this code to a file named `01-hello.js` inside `src` directory. 
 ## Using, testing a contract
 
-Agoric contracts are typically tested using the [ava](https://github.com/avajs/ava) framework. The test file begins with an `import @endo/init` to establish a [Hardened JavaScript](../js-programming/hardened-js) environment. We also import `E()` in order to make asynchronous method calls and `test` function from `ava`. _We'll talk more about [using `E()` for async method calls](../js-programming/eventual-send) later._ Following these `import` statements, we write a simple test that validates that the `greet` method works as expected.
+Agoric contracts are typically tested using the [ava](https://github.com/avajs/ava) framework -  follow the instructions there to install `ava`. The test file begins with an `import @endo/init` to establish a [Hardened JavaScript](../js-programming/hardened-js) environment. We also import `E()` in order to make asynchronous method calls and `test` function from `ava`. _We'll talk more about [using `E()` for async method calls](../js-programming/eventual-send) later._ Following these `import` statements, we write a simple test that validates that the `greet` method works as expected.
```

Comment:
Why "follow the instructions there to install `ava`"? In our getting started flow, `ava` is installed with other dev tools in the [yarn install step](https://docs.agoric.com/guides/getting-started/#installing-dapp-dependencies).

Reply:
I thought we may not assume that they have a `packages.json` file in their working directory?

Reply:
Should I change it to `yarn ava` ?

Reply:
No `package.json`? Then how are dependencies such as `import '@endo/init';` satisfied? ES6 module `import` declarations won't even work without `"type": "module"` in `package.json`, right?

This is why I postponed details on how to run a test to the context of dapp-offer-up.

I didn't intend "contract basics" to be a tutorial where you could run the pieces. It was just a bit of background info between the getting-started tutorial and the contract walkthru.

Reply:
Yes, if we're going to give a command, `yarn ava` is a good one to give.

Reply:
@dckc Pardon my ignorance in the above comment - they will of course have a `package.json`. I have updated the said command to `yarn ava` and removed the sentence: _"follow the instructions there to install `ava`"_. Thanks for your extensive and timely input.